.github/CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 # Release 0.27.0
```

Comment:
We didn't have a version bump for 0.28 dev version.
Could you please use this PR to make that?

Reply:
Good idea! I plan to work on it in a separate PR to make the updates organizing in more clear way. How do you think?

Reply:
Feel free to open a new PR for that, but I think you can do it here if you like.

Reply:
done! Thanks